src/tests/ftest/util/ior_test_base.py

```diff
@@ -156,9 +158,14 @@ def run_ior_with_pool(self):
             self.start_dfuse()
             self.ior_cmd.test_file.update(self.dfuse.mount_dir.value
                                           + "/testfile")
+            # Run IOR with intercept flag. Only applicable to POSIX.
+            out = self.run_ior(self.get_job_manager_command(), 
+                         self.processes, intercept)
+        else:
+            # Run IOR
+            out = self.run_ior(self.get_job_manager_command(), self.processes)
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'intercept' in method call